src/runtime/less/ts-pagers.less

```diff
@@ -54,17 +53,15 @@
 				&.ts-more span:after,
 				&.ts-less span:before {
 					content: '\2026';
-					position: absolute;
 					font-weight: @ts-fontweight;
 					font-size: @ts-fontsize-mini;
 				}
 				&.ts-less span:before {
-					margin-left: -13px;
 					//display: none;
 				}
 				svg {
 					position: relative;
-					top: (-@ts-unit-half - 1px);
+					top: @ts-unit-quarter + 2;
```

Comment:
+2 or +2px?

Reply:
> +2 or +2px?

px is not necessary in less, but i should follow the style.